pages/common/perl.md

```diff
@@ -0,0 +1,23 @@
+# perl
+
+> The Perl 5 language interpreter 'CLI'.
+
+- Parse and execute a file:
+
+`perl {{file}}`
+
+- Parse and execute a statement:
+
+`perl -e {{statement}}`
+
+- Check syntax error:
```

Comment:
It should be a perl script/statement, I think this info seems to be redundant. (It's a perl-thing what were you thinking) :D Anyway, just let me know if you want to apply your point of view.